tema5/src/main/java/ro/scoalainformala/Main.java

```diff
@@ -0,0 +1,73 @@
+package ro.scoalainformala;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Person person1 =new Person(34, "John Doe");
+        Person person2 =new Person(20, "Mary Smith");
+
+        Country country1 = new Country("Romania");
+        Country country2 = new Country("Canada");
+        Country country3 = new Country("United Kingdom");
+        Country country4 = new Country("United States");
+        Country country5 = new Country("Greece");
+        Country country6 = new Country("Germany");
+
+        AddressForHobby adress1 =new AddressForHobby("Name of the first address", country1);
+        AddressForHobby adress2=new AddressForHobby("Name of the second address", country2);
+        AddressForHobby adress3 =new AddressForHobby("Name of the third address", country3);
+        AddressForHobby adress4 =new AddressForHobby("Name of the fourth address", country4);
+        AddressForHobby adress5=new AddressForHobby("Name of the fifth address", country5);
+        AddressForHobby adress6=new AddressForHobby("Name of the sixth address", country6);
+
+        List<AddressForHobby> adressesForSwimming = new ArrayList<>();
+        adressesForSwimming.add(adress1);
+        adressesForSwimming.add(adress2);
+
+        List<AddressForHobby> addressesForGolfing = new ArrayList<>();
+        addressesForGolfing.add(adress3);
+        addressesForGolfing.add(adress4);
+
+        List<AddressForHobby> addressesForRunning = new ArrayList<>();
+        addressesForRunning.add(adress5);
+        addressesForRunning.add(adress6);
+
+        Hobby swimming = new Hobby(3, "Swimming", adressesForSwimming);
+        Hobby golfing = new Hobby(1,"Golfing", addressesForGolfing);
+        Hobby running = new Hobby(2, "Running", addressesForRunning);
+
+        List<Hobby> hobbies =new ArrayList<>();
+        hobbies.add(swimming);
+        hobbies.add(golfing);
+
+        List<Hobby> hobbies2 =new ArrayList<>();
+        hobbies2.add(running);
+
+        Map<Person, List<Hobby>> people = new HashMap<>();
+
+        people.put(person1,hobbies);
+        people.put(person2,hobbies2);
+
+        for(Map.Entry<Person, List<Hobby>> entry : people.entrySet())
+        {
```

Comment:
```suggestion
        for(Map.Entry<Person, List<Hobby>> entry : people.entrySet()) {
```
try to follow java convention